client-dotnet/src/EvoMaster.Client.Util/ConcurrentHashSet.cs

```diff
@@ -0,0 +1,42 @@
+using System;
+using System.Collections;
+using System.Collections.Concurrent;
+using System.Collections.Generic;
+
+namespace EvoMaster.Client.Util
+{
+    public class ConcurrentHashSet<T> : ICollection<T>, IReadOnlyCollection<T>
```

Comment:
what is the need for this? 
why not using that `ConcurrentDictionary` directly?

Reply:
.NET doesn't have a concurrent HashSet. What I have written is nothing but a wrapper around the ConcurrentDictionary. The value is byte and always zero. I though that it might be meaningless to use a key,value structure here.